src/ui/server/controllers/insurance/your-buyer/traded-with-buyer/index.test.ts

```diff
@@ -195,6 +203,23 @@ describe('controllers/insurance/your-buyer/traded-with-buyer', () => {
           expect(res.redirect).toHaveBeenCalledWith(expected);
         });
       });
+
+      describe(`when the totalContractValueOverThreshold is true and ${TRADED_WITH_BUYER} is false`, () => {
+        it(`should redirect to ${CREDIT_INSURANCE_COVER}`, async () => {
+          req.body = {
+            [TRADED_WITH_BUYER]: 'false',
+          };
+
+          // @ts-ignore
```

Comment:
Why `ts-ignore`? Can update application types?

Reply:
it keeps insisting that res.local.applications could be undefined

Reply:
Try this?

```js
res.locals.application = {
  ...mockApplication,
  totalContractValueOverThreshold: true
}
```